target/linux/ramips/mt76x8/base-files/etc/board.d/02_network

```diff
@@ -34,6 +34,10 @@ ramips_setup_interfaces()
 		ucidef_add_switch_attr "switch0" "enable" "false"
 		ucidef_set_interface_lan "eth0"
 		;;
+	asus,rt-ac1200)
+		ucidef_add_switch "switch0" \
+			"1:lan" "2:lan" "3:lan" "4:lan" "0:wan" "6@eth0"
```

Comment:
This configuration is the same with hilink,hlk-7628n, hilink,hlk-7688a, etc. Please merge with those models.

```
	hilink,hlk-7628n|\
	hilink,hlk-7688a|\
	hiwifi,hc5861b|\
	...
	wrtnode,wrtnode2p|\
	wrtnode,wrtnode2r|\
	zyxel,keenetic-extra-ii)
		ucidef_add_switch "switch0" \
			"1:lan" "2:lan" "3:lan" "4:lan" "0:wan" "6@eth0"
		;;
```

Reply:
fixed